data/apis/amberdata/beacons/amberdata sol_usd.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AmberData SOL/USD",
+  "description": "AmberData - SOL/USD - Pair",
+  "beaconId": "0x6a0bf0bdefbb367c18c4281ee3ef7bd145f27daf0eec9934ab831fce90705a59",
```

Comment:
@Ashar2shahid I'm trying to get up to speed with setting up this files but I'm not able to find a reference to this `beaconId` anywhere. Shouldn't this be in the airkeeper.json file?

These are the beconIds that I was able to find for sol/usd:
0x6b7aa718a7d8bbe38c1edd0d6a59ad266a7051ef1dea3199b04db817646aa6bb
0xe2edb0e07d8fc42dd37dcbc3ff9ea15786c5dae93669ead11ed044a02c85d6e1

I'm I not getting this right?

Reply:
beaconIds are not used in the airkeeper.json file, only the subscriptionIds are. I don't think you will find a reference to beaconIds anywhere

Reply:
My bad, I meant `subscriptionId`

Reply:
> I'm I not getting this right?

do you mean the subscriptionIds are wrong ?

Reply:
Nevermind, I got confused because I thought that `subscriptions` in airkeeper.json had `beaconId` as a prop